src/Lecture1.hs

```diff
@@ -54,7 +55,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Integer -> Integer -> Integer
+sumOfSquares x y = x ^ (2 :: Int) + y ^ (2 :: Int)
```

Comment:
This is the correct solution 👍🏻 I agree that it looks a bit ugly though 😕 

Another approach is to use `*` in this particular case to avoid ambiguity:

```suggestion
sumOfSquares x y = x * x + y * y
```